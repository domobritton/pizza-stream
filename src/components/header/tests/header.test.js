import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../';

describe('Header', () => {
  test('snapshot', () => {
    expect(
      renderer
        .create(<Header />)
    ).toMatchSnapshot();
  });
});
