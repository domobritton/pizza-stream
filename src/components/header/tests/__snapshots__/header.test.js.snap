// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<header
  className="css-joc37h"
>
  <img
    alt="Pizzastream"
    className="css-1cb43wu"
    src="logo.svg"
  />
</header>
`;
